client/src/components/userInfo/userInfo.module.scss

```
@@ -0,0 +1,19 @@
+.wrapper {
+    width: 195px;
+    display: flex;
+    flex-flow: column nowrap;
+    position: relative;
+}
+
+.user_menu {
+    width: 195px;
+    position: absolute;
+}
+
+.top {
+    bottom: 67px;
```

둘 사이 간격 10px 차이나게 조정 부탁드려요 `.bottom`도 같이
```suggestion
    bottom: 60px;
```